index.js

```diff
@@ -0,0 +1,56 @@
+window.document.querySelector('#note-form').addEventListener('submit', function(event) {
+    event.preventDefault(); 
+
+    const title = window.document.querySelector("#note-title").value;
+    const content = window.document.querySelector("#note-content").value;
+    const priority = window.document.querySelector('#note-priority').value;
+
+    const note = document.createElement('div');
+    note.classList.add('note');
+
+    const noteTitle = document.createElement('div');
+    noteTitle.classList.add('note-title');
+    noteTitle.innerText = title;
+
+    const noteContent = document.createElement('div');
+    noteContent.classList.add('note-content');
+    noteContent.innerText = content;
+
+    const notePriority = document.createElement('div');
+    notePriority.classList.add('note-priority');
+    notePriority.innerText = `Priority: ${priority.charAt(0).toUpperCase() + priority.slice(1)}`;
+
+    // Append each part to the note container
+    note.appendChild(noteTitle);
+    note.appendChild(noteContent);
+    note.appendChild(notePriority);
+
+    // Add the note to the notes container
+    document.getElementById('notes-container').appendChild(note);
+
+    // Reset the form
+    document.getElementById('note-form').reset();
+});
+
+
+// Detectar la preferencia
+function applyTheme() {
```

Comment:
Why are you implementing this method? 🤔

Is it necessary?

Is it clear for you how does`@media (prefers-color-scheme: dark) {}` defined in `styles.css` work?